code/modules/cargo/packs.dm

```diff
@@ -543,6 +543,26 @@
 					/obj/item/ammo_box/magazine/wt550m9)
 	crate_name = "auto rifle ammo crate"
 
+/datum/supply_pack/security/armory/deltaelite
```

Comment:
make it /datum/supply_pack/security/deltaelite so you can open them with normal security access

Reply:
Can you open Autorifle crates with security access?

Reply:
Eh... y'know, why not?

Reply:
you can open lasers

Reply:
Sure that's a bit irrelevant.